### PR TITLE
Querqy decorations with aggs

### PR DESCRIPTION
**Pull request by Paul-Blanchaert**
@renekrie 
Can you set version for next release? Thanks

### CODE REVIEW

File: src/main/java/querqy/elasticsearch/aggregation/InternalDecorationAggregation.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        if (in.getVersion().before(Version.V_7_8_0)) {
+            aggregations = singletonList(in.readGenericValue());
+        } else {
+            aggregations = in.readList(StreamInput::readGenericValue);
+        }
+    }
+
+    @Override
+    protected void doWriteTo(final StreamOutput out) throws IOException {
+        if (out.getVersion().before(Version.V_7_8_0)) {
```
Comment (renekrie) [inline src/main/java/querqy/elasticsearch/aggregation/InternalDecorationAggregation.java:39]: I don't know if we need this as this Querqy version doesn't support ES 7.8.0 and lower at all. Do we really need this check?

**Review by renekrie — COMMENTED** [review]
Thank you - just a question on the version check (see inline comment)

File: src/main/java/querqy/elasticsearch/aggregation/InternalDecorationAggregation.java (hunk 1 above)
Comment (Paul-Blanchaert) [reply src/main/java/querqy/elasticsearch/aggregation/InternalDecorationAggregation.java:39]: If we're not going to downgrade, we don't need it...
You prefer to have it removed?

Comment (renekrie) [reply src/main/java/querqy/elasticsearch/aggregation/InternalDecorationAggregation.java:39]: Yes please!

Comment (Paul-Blanchaert) [reply src/main/java/querqy/elasticsearch/aggregation/InternalDecorationAggregation.java]: done

**Comment by renekrie** [comment]
Thank you @Paul-Blanchaert

**Pull request merged**